config/locales/en.yml

```diff
@@ -0,0 +1,46 @@
+en:
```

Comment:
study about i18n-tasks gem and run `i18n-tasks health` [check documentation here](https://github.com/glebm/i18n-tasks#usage)

Reply:
thanks. checked it out and played around with it for some time